test/functional/fixtures/es_archiver/discover/data.json

```diff
@@ -90,7 +90,7 @@
   "type": "doc",
   "value": {
     "id": "config:8.0.0",
-    "index": ".kibana",
+    "index": ".kibana_1",
```

Comment:
Hurray this last commit fixed the tests! is it really necessary to choose `.kibana_1` instead of the alias? could lead to a potential troubles in the future

Reply:
I honestly can't reproduce this failure locally, so I don't know what the issue is. Using `.kibana_1` seemed to be done in most of the other functional tests.

Reply:
i see, couldn't reproduce it locally, too

Reply:
so it's common practice, didn't know and it's fine